scale/queue/models.py

```diff
@@ -398,7 +398,7 @@ def queue_new_recipe(self, recipe_type, data, event, superseded_recipe=None, del
         :param recipe_type: The type of the new recipe to create
         :type recipe_type: :class:`recipe.models.RecipeType`
         :param data: The recipe data to run on, should be None if superseded_recipe is provided
-        :type data: :class:`recipe.data.recipe_data.RecipeData`
+        :type data: :class:`recipe.input.recipe_data.RecipeData`
```

Comment:
It looks like this shouldn't have been changed since the directory is still named `data`.